### PR TITLE
LCAM-1654|Update feature toggle to use IS_Enabled flag

### PR DESCRIPTION
**Pull request by venkat980**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/LCAM-1654)

As part of this change the logic in FeatureDecisionService is updated to use the newly added IS_ENABLED column on the FEATURE_TOGGLE table. This allows feature toggles that apply to all users to be disabled on a user-specific basis.

### CODE REVIEW

File: maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/FeatureDecisionServiceTest.java [hunk 1]
```diff
@@ -118,6 +119,40 @@ void givenUserHasFeatureToggleWithTheCorrectAction_whenFeatureToggleMethodIsInvo
         Assertions.assertTrue(result);
     }
 
+    @ParameterizedTest
+    @MethodSource("userHasFeatureToggleWithRequiredAction")
```
Comment (Mhowell494) [inline maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/FeatureDecisionServiceTest.java:123]: Instead of adding a new test that largely duplicates the existing one with just one line changed, it would be cleaner and more maintainable to update the `userHasFeatureToggleWithRequiredAction` method source.

**Review by Mhowell494 — CHANGES_REQUESTED** [review]

File: maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/FeatureDecisionServiceTest.java (hunk 1 above)
Comment (venkat980) [reply maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/FeatureDecisionServiceTest.java]: Updated.

**Review by Mhowell494 — APPROVED** [review]
